src/models/users.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+export interface IUser extends Document {
+  name: string;
+  email: string;
+  password: string;
+  isDisabled: boolean;
+  createdDate: Date;
+  updatedDate: Date;
+}
+
+const UserSchema: Schema = new Schema({
+  name: { type: String, required: true },
+  email: { type: String, required: true },
```

Comment:
Make email field unique to avoid duplicate emails

  email: { type: String, required: true, unique: true },

Reply:
Done